setup/main.go

```diff
@@ -0,0 +1,1653 @@
+				Message: "Set auto_explain.log_buffers to (will be saved to Postgres):",
+				Help:    "Include BUFFERS usage information when a plan is logged",
+				Options: getOptsWithRecommendation(logBuffersOpts, 0),
+			}, &logBuffersIdx)
+			if err != nil {
+				return err
+			}
+			logBuffers := logBuffersOpts[logBuffersIdx]
+			err = applyConfigSetting("auto_explain.log_buffers", logBuffers, state.QueryRunner)
+			if err != nil {
+				return err
+			}
+
+			var logTimingIdx int
+			var logTimingOpts = []string{"on", "off"}
+			err = survey.AskOne(&survey.Select{
+				Message: "Set auto_explain.log_timing to (will be saved to Postgres):",
+				Help:    "Include timing information for each plan node when a plan is logged; can have high performance impact",
+				Options: getOptsWithRecommendation(logTimingOpts, 1),
+			}, &logTimingIdx)
+			if err != nil {
+				return err
+			}
+			logTiming := logTimingOpts[logTimingIdx]
+			err = applyConfigSetting("auto_explain.log_timing", logTiming, state.QueryRunner)
+			if err != nil {
+				return err
+			}
+
+			var logTriggersIdx int
+			var logTriggersOpts = []string{"on", "off"}
+			err = survey.AskOne(&survey.Select{
+				Message: "Set auto_explain.log_triggers to (will be saved to Postgres):",
+				Help:    "Include trigger execution statistics when a plan is logged",
+				Options: getOptsWithRecommendation(logTriggersOpts, 0),
+			}, &logTriggersIdx)
+			if err != nil {
+				return err
+			}
+			logTriggers := logTriggersOpts[logTriggersIdx]
+			err = applyConfigSetting("auto_explain.log_triggers", logTriggers, state.QueryRunner)
+			if err != nil {
+				return err
+			}
+
+			var logVerboseIdx int
+			var logVerboseOpts = []string{"on", "off"}
+			err = survey.AskOne(&survey.Select{
+				Message: "Set auto_explain.log_verbose to (will be saved to Postgres):",
+				Help:    "Include VERBOSE EXPLAIN details when a plan is logged",
+				Options: getOptsWithRecommendation(logVerboseOpts, 0),
+			}, &logVerboseIdx)
+			if err != nil {
+				return err
+			}
+			logVerbose := logVerboseOpts[logVerboseIdx]
+			err = applyConfigSetting("auto_explain.log_verbose", logVerbose, state.QueryRunner)
+			if err != nil {
+				return err
+			}
+		}
+
+		var logFormatIdx int
+		var logFormatOpts = []string{"text", "json"}
+		err = survey.AskOne(&survey.Select{
+			Message: "Set auto_explain.log_format to (will be saved to Postgres):",
+			Help:    "Select EXPLAIN output format to be used (only text and json are supported; text format is currently experimental)",
+			Options: getOptsWithRecommendation(logFormatOpts, 1),
+		}, &logFormatIdx)
+		if err != nil {
+			return err
+		}
+		logFormat := logFormatOpts[logFormatIdx]
+		err = applyConfigSetting("auto_explain.log_format", logFormat, state.QueryRunner)
+		if err != nil {
+			return err
+		}
+
+		var logMinDuration int
+		err = survey.AskOne(&survey.Input{
+			Message: "Set auto_explain.log_min_duration, in milliseconds, to (will be saved to Postgres):",
```

Comment:
I wonder if we should just default to 1000 here? (to get a working setup without the person setting up wondering whats a good threshold - which they likely don't know yet, because they could evaluate this based on the query performance data they can only access at a later point)

Reply:
Like, do a two-part question like I'm doing in other places (1000/other, then enter other if that's what you selected) or just do yes/no set-to-1000 (and you can tweak separately if you want)?

Reply:
Maybe 1000/other is good (with 1000 being recommended) - if this is the Nth server someone is setting up they might know exactly which value they'd like to use.